doc/config.rst

```diff
@@ -297,3 +297,102 @@ recommended instead to use a *secure* local repository:
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| ``ar``                | Usually provided by GHC's ``"ar command"`` entry in ``ghc --info``. Note this might refer to ``llvm-ar`` instead of GNU's ``ar``.  |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| ``c2hs``              | `<https://hackage.haskell.org/package/c2hs>`_                                                                                      |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| ``doctest``           | `<https://hackage.haskell.org/package/doctest>`_                                                                                   |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| ``gcc``               | Usually provided by GHC's ``"C compiler command"`` entry in ``ghc --info``. Note this might refer to ``clang`` instead of ``gcc``. |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| ``ghc``               |                                                                                                                                    |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| ``ghc-pkg``           |                                                                                                                                    |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| ``ghcjs``             |                                                                                                                                    |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| ``ghcjs-pkg``         |                                                                                                                                    |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| ``greencard``         | `<https://hackage.haskell.org/package/greencard>`_                                                                                 |
```

Comment:
`greencard` doesn't appear to have been updated since 2014.

Reply:
You mean I should probably add some comment there saying that it is probably an outdated tool, right?

Reply:
Yes, as with `haskell-suite`.